modules/ding_ting_frontend/ding_ting_frontend.info

```diff
@@ -19,7 +19,7 @@ dependencies[] = page_manager
 dependencies[] = pm_existing_pages
 dependencies[] = strongarm
 dependencies[] = ting
-dependencies[] = ting_covers
```

Comment:
I see what you mean.

Just noticed that ting_covers_addi is activated on update in ting_covers, so wouldn't it be better to also ativate it on install in ting_covers?

Also the update hook which activates it checks if the variable addi_wsdl_url is set. This will never be set on a fresh install, so should we just activate it anyway?